src/app/core/common-components/entity-form/entity-form.service.ts

```diff
@@ -188,7 +188,7 @@ export class EntityFormService {
         newVal = new Date();
         break;
       case PLACEHOLDERS.CURRENT_USER:
-        newVal = this.currentUser.value.getId();
+        newVal = this.currentUser.value.getId(true);
```

Comment:
can we add a safeguard here to gracefully handle the currentUser not being defined? (as is nowadays often the case for our support account)